src/client.ts

```diff
@@ -351,7 +360,7 @@ export class Client {
             capabilities: this.activeWallets.external.capabilities,
           },
         },
-        ephemeralPub: this.ephemeralPub.getPublic().encode('hex'),
+        ephemeralPub: this.ephemeralPub?.getPublic()?.encode('hex'),
```

Comment:
Please make sure errors are handled whenever this get called, since it will not work if it has not connected to a device. It may be that such a case is functionally impossible now that you require a device ID, but pls take a look

Reply:
This is in a try/catch, and the `?` makes it less likely to throw. 

The required-ness of `deviceId` in the `Client` class shouldn't have changed.